olxcleaner/objects/common.py

```diff
@@ -105,7 +105,7 @@ def is_pointer(self, attribs=None):
         if attribs is None:  # pragma: no cover
             attribs = self.attributes
         # Follows pointer convention in edX: is_pointer_tag in
-        # https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/xml_module.py
+        # https://github.com/openedx/edx-platform/blob/master/common/lib/xmodule/xmodule/xml_module.py
```

Comment:
Link should be https://github.com/openedx/edx-platform/blob/master/xmodule/x_module.py, now that xmodule lives in a top level directory.